tests/app/dao/test_service_contact_list_dao.py

```diff
@@ -0,0 +1,14 @@
+from app.dao.service_contact_list_dao import dao_get_contact_lists
+from tests.app.db import create_service_contact_list
+
+
+def test_dao_get_contact_lists(notify_db_session):
+    contact_list = create_service_contact_list()
+    fetched_list = dao_get_contact_lists(contact_list.service_id)
+
+    assert len(fetched_list) == 1
+    assert fetched_list[0] == contact_list
+
+
+def test_save_contact_lists(sample_service):
+    pass
```

Comment:
think we can remove this test stub - it's for a boring function, that we'll test properly through test_rest.py